cloud/jenkins/pgo_operator_eks_latest.groovy

```diff
@@ -440,36 +440,24 @@ pipeline {
         stage('Run Tests') {
             parallel {
                 stage('cluster1') {
-                    agent {
```

Comment:
I am not sure that it is a  good idea because all tests will be run on the same agent in parallel and you will have a lot of conflicts

Reply:
I am experimenting to see why it is needed to have separate blocks of code like this:
```
                    agent {
                        label 'docker'
                    }
```
in every stage V.S. having a common block in the beginning, like this:
```
    agent {
        label 'docker'
    }
```
like all the rest of the pipelines for the other operators. It looks like a copy/paste error for me, but I want to be sure.

Reply:
OK, it looks like PG and PS use new node for each parallel stage - it is some limitation of kuttl. The presence of
```
                    agent {
                        label 'docker'
                    }
```
makes the new node. You are right about this, I will fix it.